images/win/scripts/Installers/Install-JavaTools.ps1

```diff
@@ -85,14 +81,8 @@ setx MAVEN_OPTS $maven_opts /M
 $uri = 'https://ayera.dl.sourceforge.net/project/cobertura/cobertura/2.1.1/cobertura-2.1.1-bin.zip'
 $coberturaPath = "C:\cobertura-2.1.1"
 
-cd $env:TEMP
-
-Invoke-WebRequest -UseBasicParsing -Uri $uri -OutFile cobertura.zip
-
-# Expand the zip
-Expand-Archive -Path cobertura.zip -DestinationPath "C:\" -Force
-
-# Deleting zip folder
-Remove-Item -Recurse -Force cobertura.zip
+Start-DownloadWithRetry -Url $uri -Name "cobertura.zip" |
+Expand-Archive -Path {$_} -DestinationPath "C:\" |
+Remove-Item -Recurse -Force -Path {$_}
```

Comment:
What do we want to delete?

Reply:
We should delete `.zip` file after expanding, but I think if we downloaded this file to `$env:TEMP` directory, removing is not necessary.